susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1142,6 +1156,11 @@
         'DEST' : '/srv/www/htdocs/pub/repositories/ubuntu/18/4/bootstrap/',
         'TYPE' : 'deb'
     },
+    'ubuntu-20.04-amd64-uyuni' : {
```

Comment:
Missing from this PR: the SUSE Manager repo. Can't add them until SCC channels for Ubuntu 20.04 are ready.